nodesharing-lib/pom.xml

```diff
@@ -21,6 +21,55 @@
             <groupId>com.google.code.gson</groupId>
             <artifactId>gson</artifactId>
         </dependency>
+
+       <dependency>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>credentials</artifactId>
+        <!-- <version>2.1.16</version> -->
+      </dependency>
```

Comment:
Please do not move the plugin dependencies into the `nodesharing-lib` - even when both `plugin` and `backend-plugin` needs the same plugins. The purpose of the library is to share code that does not use these dependencies.

Reply:
Taken care of this. Thank you